pom.xml

```diff
@@ -18,28 +18,71 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.eclipse.microprofile.parent</groupId>
+    <groupId>org.eclipse.microprofile</groupId>
     <artifactId>microprofile-parent</artifactId>
-    <version>1-SNAPSHOT</version>
+    <version>1.0-SNAPSHOT</version>
+
     <packaging>pom</packaging>
-    <name>MicroProfile Parent</name>
-    <description>Parent POM for Eclipse MicroProfile</description>
+    <name>MicroProfile Parent POM</name>
     <url>http://microprofile.io</url>
 
     <properties>
+        <!-- Cross plugins settings -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <!-- Compiler -->
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <checkstyle.version>2.17</checkstyle.version>
-        <checkstyle.methodNameFormat>^_?[a-z][a-zA-Z0-9]*$</checkstyle.methodNameFormat>
+
+        <!-- Dependencies - Jakarta -->
+        <dependency.jakarta.api.version>8.0.0</dependency.jakarta.api.version>
```

Comment:
Can you create a `configurations` element to store all of the version configuration so that it is much easier to maintain?

Reply:
ignore my previous comment. I meant what you are doing here.

Reply:
I realised that I was confused with the prefix of `dependency`. Can we remove that prefix as it does not add any value?

Reply:
I think it works great with the IDE and auto completion, but I'll change it.